lib/event_planner.rb

```diff
@@ -0,0 +1,161 @@
+require "./games"
```

Comment:
Careful with this across the board. Expectation with `require` is that you run code from the project root directory. Looks like you were using `cd test` to get into your test directory and `cd lib` to get into your lib directory to run code. Instead of actually moving to those directories, run your commands from the project root like this:

```
ruby test/game_test.rb
ruby lib/runner.rb
```